lib/text/ui_text_displayer.js

```diff
@@ -504,6 +508,19 @@ shaka.text.UITextDisplayer = class {
       regionElement.style.left = region.viewportAnchorX -
         region.regionAnchorX * region.width / 100 + viewportAnchorUnit;
     }
+    if (region.heightUnits !== pixelUnit &&
+        region.widthUnits !== pixelUnit &&
+        region.viewportAnchorUnits !== pixelUnit) {
+      // Clip region
```

Comment:
The PR description and comment and such say that this is clipping the region.
Technically speaking though, "clipping" the region would mean making it smaller to fit into the window. This is offsetting the region instead, which is a different operation.
Now, offsetting is probably the better operation in this case, since otherwise we'll be trying to fit text into tiny regions. But it'd be worth renaming the PR and changing the comment.

Reply:
Done!